nfsstat/check.py

```diff
@@ -0,0 +1,129 @@
+# (C) Datadog, Inc. 2010-2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# stdlib
+import os
+
+# 3rd party
+
+# project
+from checks import AgentCheck
+from utils.subprocess_output import get_subprocess_output
+
+EVENT_TYPE = SOURCE_TYPE_NAME = 'nfsstat'
+
+class NfsStatCheck(AgentCheck):
+
+    metric_prefix = 'system.nfs.'
+
+    def check(self, instance):
+        # if they set the path, use that
```

Comment:
Little nit, and this seems to be prevalent in a lot of our checks, so it may be by design... But, why redo the check for `nfs_cmd` on every check (i.e. every 15 secs).  There's not a lot of overhead, but it is wasted nonetheless (unless we're expecting that to change from one run to the next?

Reply:
For this one, no, there should be only one instance allowed per box, so we can set it in the init_config and store it in the object. Also, there's probably only one instance allowed per box anyway. I'll note that.

The reason it's used in a lot of checks is because we run each check instance in the same object instance, so we have to set the proper thing each time, rather than storing it in the object.